RELEASE/scripts/autoscend/auto_mclargehuge.ash

```diff
@@ -126,7 +126,7 @@ boolean L8_trapperExtreme()
 	{
 		return false;
 	}
-	if (internalQuestStatus("questL08Trapper") < 0 || internalQuestStatus("questL08Trapper") > 2)
+	if (internalQuestStatus("questL08Trapper") < 2 || internalQuestStatus("questL08Trapper") > 2)

```

Comment:
This could be simplified to `if (internalQuestStatus("questL08Trapper") != 2)`, right?

Reply:
done :+1: